src/OAuth/AccessKey.cs

```diff
@@ -20,5 +22,12 @@ public class AccessKey
 
         [JsonProperty("jwk")]
         public JsonWebKey Jwk { set; get; }
+
+        public static AccessKey CreateFromBase64EncodedAccessKey(string encoded)
```

Comment:
Can you add unit tests for this function?

Reply:
sure, that's a good idea, i guess i'll probably also add them for the other 2 corresponding projects

Reply:
so after some thought i think you pretty much covered all the paths for the unit tests

![image](https://user-images.githubusercontent.com/103658932/182885443-95e551a7-c550-42c9-84a2-4e32aa685fd4.png)

Reply:
ok so i did end up adding some additional test cases with special characters along with the rest, not sure if it is needed tho

Reply:
oh and also, for the sample accesskey/base64 string, i did remember to use a fake accesskey